recipes/boost/all/conanfile.py

```diff
@@ -146,6 +148,8 @@ class BoostConan(ConanFile):
         "buildid": None,
         "python_buildid": None,
         "system_use_utf8": False,
+        "without_math_c99l": False,
```

Comment:
How does it work, do they need Boost Math?

Reply:
Yes, they do: https://github.com/conan-io/conan-center-index/blob/master/recipes/boost/all/dependencies/dependencies-1.81.0.yml#L91

@hoyhoy please, remove these options in case boost math is disabled. You can do it in the `configure()` method.